stock_quant_manual_assign/README.rst

```diff
@@ -32,6 +32,12 @@ This module allows you to manually change the automatic quant selection.
 .. contents::
    :local:
 
+
+Configuration
+=============
+
+On /Inventory/Configuration/Operation Types you can configure to set quantity done or not on manual quants assign.
```

Comment:
You mustn't edit this file, but add the fragment `CONFIGURE.rst` on `readme` folder, and generate the new README or let the bot to generate it on merge.

Reply:
FIXED